Makefile

```
@@ -857,6 +857,8 @@ else
 endif
 THIRD_PARTY_SOURCES := $(wildcard $(SRC_DIR)/third-party/flatbuffers/*.cpp)
 HEADERS := $(wildcard $(SRC_DIR)/*.h)
+OBJECT_CREATOR_SOURCES := $(wildcard $(SRC_DIR)/object_creator/*.cpp)
+OBJECT_CREATOR_HEADERS := $(wildcard $(SRC_DIR)/object_creator/*.h)
```

`SRC_DIR` is src
```suggestion
OBJECT_CREATOR_SOURCES := $(wildcard $object_creator/*.cpp)
OBJECT_CREATOR_HEADERS := $(wildcard $object_creator/*.h)
```
or `$(SRC_DIR)/../object_creator/`